### ISSUE TITLE
Can't MCU connection dfu mode, MCU USB not working.

### ISSUE THREAD

**Issue by pure100kim** [makerbase-mks/MKS-SKIPR#19]
I connected usb to mcu type-c port. and then
pressed the boot0 switch and turned on the power button, but the dfu usb is not recognized at all.
It is not recognized as STM32CUBE. Not even ST-LINK.

**Comment by MKS-hosiey** [comment]
Is your data cable connected? There are two type_c interfaces on the motherboard, you need to connect to the MCU interface

**Comment by pure100kim** [comment]
> Is your data cable connected? There are two type_c interfaces on the motherboard, you need to connect to the MCU interface

Of course. I used data cable including your acc in the box. And also used cell phone data cable.
